### PR TITLE
deployment

### PR DESCRIPTION
**Pull request by Nazarii77**

### CODE REVIEW

File: src/app/components/main-page/main-page.component.html [hunk 1]
```diff
@@ -35,13 +35,13 @@ <h3 class="centered">Top rental bikes</h3>
         <span class="card__category"> ₴ 100 / day</span> <br>
         <span class="card__category"> ₴ 400 / week</span>
```
Comment (YarynaN) [inline src/app/components/main-page/main-page.component.html]: we do not have implemented logic for discounts.

File: src/app/components/main-page/main-page.component.html [hunk 2]
```diff
@@ -51,7 +51,7 @@ <h3 class="card__title">Trek Marlin </h3>
         <span class="card__category"> ₴ 200 / day</span> <br>
         <span class="card__category"> ₴ 600 / week</span>
         <h3 class="card__title">Ghost Rocket 2</h3>
```
Comment (YarynaN) [inline src/app/components/main-page/main-page.component.html]: we do not have implemented logic for discounts.

File: src/app/components/main-page/main-page.component.html [hunk 3]
```diff
@@ -67,7 +67,7 @@ <h3 class="card__title">Ghost Rocket 2</h3>
         <span class="card__category"> ₴ 100 / day</span> <br>
         <span class="card__category"> ₴ 500 / week</span>
```
Comment (YarynaN) [inline src/app/components/main-page/main-page.component.html]: we do not have implemented logic for discounts.

**Review by YarynaN — APPROVED** [review]

File: src/app/components/main-page/main-page.component.html (hunk 1 above)
Comment (Nazarii77) [reply src/app/components/main-page/main-page.component.html]: I know it is for cool look

Comment (YarynaN) [reply src/app/components/main-page/main-page.component.html]: @Nazarii77 do you insist on keeping it?

Comment (Nazarii77) [reply src/app/components/main-page/main-page.component.html]: yes, we presented with this one today

**Review by NatiKuz — APPROVED** [review]

**Pull request merged**